layout/Library/Application Support/iSponsorBlock.bundle/th.lproj/Localizable.strings

```diff
@@ -0,0 +1,86 @@
+/* iSponsorBlock การตั้งค่า */
+"Enabled" = "เปิดออก";
+"RestartFooter" = "โปรดรีสตาร์ท YouTube เพื่อใช้การเปลี่ยนแปลง";
+
+"Sponsor" = "ผู้สนับสนุนโฆษณา";
+"Intermission/IntroAnimation" = "มิดฟิลด์/เปิดแอนิเมชัน";
+"Intermission" = "การเปลี่ยนผ่าน";
+"Endcards/Credits" = "จบการ์ด/รายการ";
+"Outro" = "ตอนจบ";
+"InteractionReminder" = "ปฏิสัมพันธ์ของผู้ชม (การสมัครสมาชิก)";
+"InteractionReminder_Subcribe/Like" = "ปฏิสัมพันธ์ของผู้ชม (การสมัครสมาชิก/ชอบ)";
+"Interaction" = "เชิงโต้ตอบ";
+"Unpaid/SelfPromotion" = "ฟรี/self -propaganda";
```

Comment:
Propaganda? That is a weird choice of translation. I'd opt for "self promotion" to be "โปรโมตตัวเอง".